rir/src/compiler/pir/instruction.h

```diff
@@ -653,6 +656,8 @@ class FLIE(LdVar, 1, Effects() | Effect::Error | Effect::ReadsEnv) {
     size_t gvnBase() const override {
         return hash_combine(InstructionImplementation::gvnBase(), varName);
     }
+
+    bool needsReferenceCount() const override { return false; }
```

Comment:
Why doesn't the `LdVar` need refcount?

Reply:
ldvar values already have a guaranteed named count of 1. ensure named will bump the refcount to 1, so it should never be neccessary for ldvar values.